shared/message/platform/webots/messages.proto

```diff
@@ -66,27 +67,33 @@ message Force6DMeasurement {  // not yet implemented
 }
 
 message CameraMeasurement {
-    string name                   = 1;
-    int32  width                  = 2;
-    int32  height                 = 3;
-    int32  quality                = 4;  // 100 = no compression, 0 = high compression
-    bytes  data                   = 5;  // JPEG encoded data
-    double horizontalFieldOfView  = 6;  // express in [rad]
-    int32  centerX                = 7;  // horizontal optical center in pixels
-    int32  centerY                = 8;  // vertical optical center in pixels
-    double radialCoefficients     = 9;
-    double tangentialCoefficients = 10;
+    string name    = 1;
+    uint32 width   = 2;
+    uint32 height  = 3;
+    sint32 quality = 4;  // -1 = raw image, 100 = no compression, 0 = high compression
+    bytes  image   = 5;  // RAW or JPEG encoded data (note: JPEG is not yet implemented)
+}
+
+message Message {
+    enum MessageType {
+        ERROR_MESSAGE   = 0;
+        WARNING_MESSAGE = 1;
+    }
+    MessageType message_type = 1;
+    string      text         = 2;
 }
 
 message SensorMeasurements {
-    double time = 1;  // time stamp at which the measurements were performed expressed in [s]
-    repeated PositionSensorMeasurement position_sensor = 2;
-    repeated AccelerometerMeasurement  accelerometer   = 3;
-    repeated GyroMeasurement           gyro            = 4;
-    repeated BumperMeasurement         bumper          = 5;
-    repeated Force3DMeasurement        force_3d        = 6;
-    repeated Force6DMeasurement        force_6d        = 7;
-    repeated CameraMeasurement         camera          = 8;
+    uint32           time     = 1;  // time stamp at which the measurements were performed expressed in [ms]
```

Comment:
`[ms]` with reference to some fixed point in time?? Simulation start time? Time that we connected to the server? Epoch time?

Reply:
Simulation time since the start of the connection with the Webots controller.